src/sets.jl

```diff
@@ -438,6 +438,38 @@ Base.isapprox(a::SOS2{T}, b::SOS2{T}; kwargs...) where T = isapprox(a.weights, b
 
 dimension(s::Union{SOS1, SOS2}) = length(s.weights)
 
+"""
+	ActivationCond
+
+Activation condition for an indicator constraint.
+Used as first type parameter of `IndicatorSet{ActiveCond,S}`.
+"""
+@enum ActivationCond begin
+    ACTIVATE_ON_ZERO
+    ACTIVATE_ON_ONE
+end
+
+"""
+    IndicatorSet{A, S <: AbstractScalarSet}(set::S)
+
+Set ``\\{(y, x) \\in \\{0, 1\\} \\times \\mathbb{R}^n``
+Such that: ``y = 0 \\implies x \\in set\\}``
```

Comment:
I'm having trouble parsing these two lines as a mathematical set. (There's text in the middle of the set definition?).
You can write this as ``\\{((y, x) \\in \\{0, 1\\} \\times \\mathbb{R}^n : y = 0 \\implies x \\in set\\}``
or also ``\\{0 \\} \\times \\mathbb{R} \cup \\{1\\} \\times set``. You should also compile this and check that the equations display reasonably in the doc.

Reply:
fixed in last commit yup. Doc built it renders well